packages/ketcher-core/src/chemistry/entities/Fragment.ts

```diff
@@ -13,30 +13,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  ***************************************************************************/
-function calcStereoFlag(struct: any, stereoAids: Array<number>) {
+export enum StereoFlag {
+  Mixed = 'Mixed',
+  abs = 'ABS (Chiral)',
+  and = 'AND Enantiomer',
+  or = 'OR Enantiomer'
+  // todo: custom in the future
+}
+
+function calcStereoFlag(struct: any, stereoAids: Array<number>): string | null {
```

Comment:
and `Struct` (from 'ketcher-core') for `struct` argument

Reply:
fixed